app/src/main/AndroidManifest.xml

```diff
@@ -14,10 +14,17 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     tools:targetApi="m">
+
     <activity
-      android:name=".splash.KiwixSplashActivity"
+      android:name=".main.KiwixMainActivity"
+      android:configChanges="orientation|keyboardHidden|screenSize"
       android:label="@string/app_name"
-      android:theme="@style/ThemeOverlay.KiwixTheme.Launcher">
```

Comment:
I have concerns over this, I know you moved this theme to the layout but I wonder if that is enough to maintain the feature we want. On a cold start of the app the user should see the kiwix logo, the theme is used to show this image before the inflation of a view even happens, [link](https://android.jlelse.eu/the-complete-android-splash-screen-guide-c7db82bce565).

This [answer](https://stackoverflow.com/a/52835946/12225251) is probably the route

Reply:
How can I test that this works?